Assets/GothicVR/Scripts/Debugging/FeatureFlags.cs

```diff
@@ -33,6 +33,8 @@ public enum SunMovementPerformance
 
         [Header("__________DayTime__________")]
         public bool enableDayTime;
+        [Tooltip("Modifies how fast the in-game time passes")]
+        [Range(0.5f, 300f)] public float TimeMultiplier;
```

Comment:
One more thing is missing: Please set this value inside FeatureFlagTool correct. So that we have proper production value for it.

Reply:
Done